src/api/apiHandler.js

```diff
@@ -1,7 +1,12 @@
-export const apiHandler = async (endpoint, method = 'GET', requestBody) => {
+export const apiHandler = async (
+    endpoint,
+    method = 'GET',
```

Comment:
no need to set 'Get' by default

Reply:
I have seen examples of doing it this way. Why do you not recomended?
This shortens 'GET' requests that do not have additional parameters, since we can only specify the API and nothing else.

Reply:
might make sense but then, i need to dive into the apiHandler and check it's logic to identify how to make 'Get ' requests, instead of copypasting it from other place of the app